active/tab/tabcontrol.md

```diff
@@ -0,0 +1,245 @@
+            TabDraggedOutside="OpenTabInNewWindow">
+    <TabControl.TabFooter>
+        <Button Content="+" Click="NewTab_Click" />
+    </TabControl.TabFooter>
+    ...
+</TabControl>
+```
+
+## Put tabs in the titlebar area
+
+![A sample app that shows the Tab control can extend into the titlebar area](images/tab-extend-to-title.png)
+
+``` xml
+<TabControl>
+    <TabItem Header="Tab 1" />
+    <TabItem Header="Tab 2" />
+    <!-- More tabs -->
+</TabControl>
+
+<Grid x:Name="CustomDragRegion" Width="200" Height="40" HorizontalAlignment="Right" VerticalAlignment="Top" />
+```
+
+``` csharp
+// Customize the titlebar area using the guidance from here: https://docs.microsoft.com/en-us/windows/uwp/design/shell/title-bar
+public MainPage()
+{
+    this.InitializeComponent();
+
+    var coreTitleBar = CoreApplication.GetCurrentView().TitleBar;
+    coreTitleBar.ExtendViewIntoTitleBar = true;
+
+    Window.Current.SetTitleBar(CustomDragRegion);
+}
+```
+
+## Create a new window when tearing out tabs
+
+![An example picture showing that a tab can be torn out and moved to its own window.](images/tab-new-window.png)
+
+See the [TabView tear out sample](https://github.com/windows-toolkit/Sample-TabView-TearOff/tree/master/TabViewTear) for a more complete sample.
+
+``` xml
+<TabControl CanDragItems="True"
+            CanReorderItems="True"
+            TabDraggedOutside="TabControl_TabDraggedOutside">
+```
+``` csharp
+// NOTE: The app is responsible for writing this code. We will provide a sample that may look something like:
+private async void TabControl_TabDraggedOutside(object sender, TabDraggedOutsideEventArgs e)
+{
+    // Create a new AppWindow
+    AppWindow newWindow = await AppWindow.TryCreateAsync();
+
+    // Create the content for the new window
+    var newPage = new MainPage();
+
+    // Remove tab from existing list
+    Tabs.Items.Remove(e.Tab);
+
+    // Add tab to list of Tabs on new page
+    newPage.AddItemToTabs(e.Tab);
+
+    // Set the Window's content to the new page
+    ElementCompositionPreview.SetAppWindowContent(newWindow, newPage);
+
+    // Show the window
+    await newWindow.TryShowAsync();
+}
+```
+
+## Databind to a set of tabs 
+
+``` xml
+<TabControl ItemsSource="{x:Bind TabItemCollection}" />
+```
+
+# Remarks
+<!-- Explanation and guidance that doesn't fit into the Examples
+section.  For example, see the Remarks for the MediaPlayerElement 
+(https://docs.microsoft.com/uwp/api/Windows.UI.Xaml.Controls.MediaPlayerElement#remarks). -->
+
+## Keyboarding behavior
+### TAB and Arrow key behavior
+The TabControl has areas containing left content, right content, and the Tab content (containing 0 or more Tabs). When the user hits the TAB key, focus moves between the content areas. When focus moves into the Tab content area, the selected Tab gains focus. Once the user is in the Tab content area, pressing the arrow key will move focus between the tabs. Arrow focus is trapped inside the Tab content area.
```

Comment:
I will follow up with @chigy about the keyboarding behavior. The behavior as spec'd matches our overall keyboarding guidance. 

Since the + button is developer provided, if we decide that the arrowing behavior suggested in Edge (dev) is the way to go, we will want to provide a sample about how to make this work.

Reply:
Typical tab control behavior is to switch tabs with Control+Tab, but the unmodified tab key moves keyboard focus like most other cases.